db/migrations/00005_create_storage_diffs_table.sql

```diff
@@ -7,6 +7,7 @@ CREATE TABLE public.storage_diff
     hashed_address BYTEA,
     storage_key    BYTEA,
     storage_value  BYTEA,
+    checked        BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
I thought about having multiple columns here to distinguish whether we're marking a diff as unwatched vs successfully transformed. In theory, that could facilitate adding new transformers down the line, if we could target previously unwatched diffs for the new transformer (whereas the approach in this PR would require re-checking all diffs marked as `checked`). However, I didn't invest a lot of time thinking through how that would actually work and felt like it might be more complexity that we need right now. Happy to reconsider if folks have thoughts!

Reply:
Pushing that decision off makes sense to me. I wonder if we could add a story to the backlog so we don't forget to thing that through at some point.

Reply:
👍 VDB-1190